.github/CODEOWNERS

```diff
@@ -11,30 +11,31 @@
 # Packages
```

Comment:
Other than adding yourself to this file, please do not make any other changes to the file.

Reply:
FYI: Second off the errors where there before I even added myself too the list.
The last commit had errors
`This CODEOWNERS file contains errors …
CODEOWNERS errors
Unknown owner on line 27: make sure @moebius87 exists and has write access to the repository
*/k9s*                @moebius87
Unknown owner on line 28: make sure @johnstevenson exists and has write access to the repository
*/composer*           @johnstevenson
Unknown owner on line 29: make sure @jetersen exists and has write access to the repository
*/krew*               @jetersen
Unknown owner on line 30: make sure @jetersen exists and has write access to the repository
*/kubelogin*          @jetersen
Unknown owner on line 31: make sure @VuiMuich exists and has write access to the repository
*/ffmpeg-full*        @VuiMuich
Unknown owner on line 32: make sure @dgalbraith exists and has write access to the repository
*/etcd*               @dgalbraith
Unknown owner on line 33: make sure @dgalbraith exists and has write access to the repository
*/octave.portable*    @dgalbraith
Unknown owner on line 34: make sure @hansnull exists and has write access to the repository
*/selenium-gecko-driver* @hansnull @chocolatey-community/chocolatey-team-maintainers
@@ -34,6 +34,7 @@
*/selenium-gecko-driver* @hansnull @chocolatey-community/chocolatey-team-maintainers
*/es*                 @TheCakeIsNaOH
*/selenium-chrome-driver* @chocolatey-community/chocolatey-team-maintainers
*/intunewinapputil*   @chocolatey-community/chocolatey-team-maintainers`

How can alphabetizing the list of packages in the file be a major change?

Reply:
> FYI: Second off the errors where there before I even added myself to the list.
> The last commit had errors

I am aware of that, this is due to GitHub not actually supporting automatic review requests from people that do not have write access to the repository.

> How can alphabetizing the list of packages in the file be a major change?

I never said it was a major change, I only said to not make any other changes than adding yourself to the list.

Reply:
Will revert as time allows.
I will hope this gets fixed before this become mandatory.

I request that the list of packages in this file be alphabetized after this PR.